Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -13,5 +13,10 @@
 
 const NativeEventEmitter = require('NativeEventEmitter');
 const KeyboardObserver = require('NativeModules').KeyboardObserver;
+var dismissKeyboard = require('dismissKeyboard');
 
-module.exports = new NativeEventEmitter(KeyboardObserver);
+let keyboard = new NativeEventEmitter(KeyboardObserver);
```

Comment:
`const` just creates an immutable binding to the variable. The value is still mutable.